TYPO3.Flow/Documentation/TheDefinitiveGuide/PartIII/Http.rst

```diff
@@ -212,6 +212,16 @@ You can, in theory, create a new ``Request`` instance by simply using the ``new`
 arguments to the constructor. However, there are two static factory methods which make life much easier. We recommend
 using these instead of the low-level constructor method.
 
+.. warning::
+
+	You should only create a ``Request`` manually if you want to send out requests or if you know exactly what you are
+	doing. Creating a new ``Request`` will completely bypass all ``HTTP Components`` and might therefore lead to
```

Comment:
That sounds a bit dramatic, just *creating*  a `Request` won't have any side effects ;) Maybe we should put it a bit more neutral?